File: tarea 2/c++/ejercicio 4.cpp
// 4. Dibuje Rectangulo
#include <iostream>
using namespace std;

int main() {
    int ancho, alto;

    cout << "Ingrese el ancho: ";
    cin >> ancho;

    cout << "Ingrese el alto: ";
    cin >> alto;

    for (int i = 1; i <= alto; i++) {
        for (int j = 1; j <= ancho; j++) {
            if (i == 1 || i == alto || j == 1 || j == ancho)
                cout << "*";
            else
                cout << " ";
        }
        cout << endl;
    }

    return 0;
}

File: tarea 2/python/ejercicio_4.py
# 4. Dibujar Rectangulo

ancho = int(input("Ingrese el ancho: "))
alto = int(input("Ingrese el alto: "))

for i in range(alto):
    for j in range(ancho):
        if i == 0 or i == alto - 1 or j == 0 or j == ancho - 1:
            print("*", end="")
        else:
            print(" ", end="")
    print()
    input(" ")